frontend/src/container/Publications.jsx

```diff
@@ -347,6 +361,7 @@ export class Publications extends React.Component {
         break
     }
     this.setState({ loading: false })
+    this.setState({ scrollTop: scrollTop })
```

Comment:
while calling multiple setState to update different properties works, it is better to update them in a single setState.
At runtime it isn't supposed to change anything since setState is asynchronous and react batch multiple setState to rerender the component once.
But having multiple setState is more error prone, therefore, the good practice is to process every required new values and call setState once.

```jsx
this.setState({
  loading: false,
  scrollTop: scrollTop
})
```